tests/acceptance/blocking_adapter_test.py

```diff
@@ -476,13 +469,13 @@ def test(self):
         self.assertIsInstance(method, pika.spec.Basic.GetOk)
         self.assertEqual(method.delivery_tag, 1)
         self.assertFalse(method.redelivered)
-        self.assertEqual(method.exchange, _p3_as_bytes(''))
-        self.assertEqual(method.routing_key, q_name)
+        self.assertEqual(method.exchange, as_bytes(''))
+        self.assertEqual(method.routing_key, as_bytes(q_name))
```

Comment:
From what I've been able to learn up till now, I think it is and as such your statement should be correct. There is one interesting thing though: pika seems to be able to pass properties in the message header for the body, such as "content-type". Now if one were to choose a standard encoding for unicode and send this information as the content-type property...